packages/terra-pills/translations/en-US.json

```
@@ -0,0 +1,12 @@
+{
+  "Terra.pills.label.delete": "Delete {pillLabelName}",
```

After doing reviews for the translations, based on researching, I think the only one we need to update is:
```suggestion
  "Terra.pills.label.delete": "Remove {pillLabelName}",
```
 
All the others seem to match closely with industry patterns, based on a few examples like these:

1. Angular a11y improvements
https://material.angularjs.org/latest/demo/chips
and https://github.com/angular/material/issues/2618
and https://github.com/angular/material/pull/11501 
and https://github.com/angular/material/pull/11422/files

2. Microsoft Fluent improvements
https://developer.microsoft.com/en-us/fluentui#/controls/web/pickers
and https://github.com/microsoft/fluentui/issues/14776

3. Suggestions via Dennis Lembree
https://weboverhauls.github.io/demos/chips/